bin/.profile.d/sigsci-agent.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell.exe -ExecutionPolicy Unrestricted -File "/Users/bbucher-sigsci/work/sigsci-cloudfoundry-buildpack/bin/.profile.d/sigsci-agent.ps1"
```

Comment:
Should this be 
`powershell.exe -ExecutionPolicy Unrestricted -File "%~dp0\sigsci-agent.ps1"`